tools/serverpod_cli/lib/src/test_util/builders/class_definition_builder.dart

```diff
@@ -320,4 +325,48 @@ class ClassDefinitionBuilder {
     _isException = isException;
     return this;
   }
+
+  ClassDefinitionBuilder withChildClasses(
```

Comment:
Suggestion: The builder method indicates it should be possible to define multiple child classes.

Please take in a list of classDefinitions instead and wrap these in ResolvedInheritanceDefinition.